### ISSUE TITLE
Error visualization does not appear when clicking eye icon

### ISSUE THREAD

**Issue by MichaelMauderer** [enso-org/ide#1595]
### What did you do?
1. Created a node with error
2. Hovered the node
3. Clicked the eye icon to see the error visualization

### What did you expect to see?
The error visualization should appear.

### What did you see instead?
Nothing appeared.

### Enso Version
678007c

### Additional notes

https://user-images.githubusercontent.com/1428930/120639980-418ce280-c472-11eb-8cc9-4bc5f061c671.mp4

**Comment by farmaazon** [comment]
The question is if it should. As I understood when discussing https://github.com/enso-org/ide/issues/1302 the error visualization should indeed be shown only on mouse hover. Remember, that from the user perspective, the error is not a visualization: it is just an error message.

**Comment by MichaelMauderer** [comment]
Agree, the other optional solution to this problem here is to hide the eye icon if there is an error to avoid confusion.
